AWS/delete_iam_user.ipynb

```diff
@@ -0,0 +1,689 @@
+{
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "metadata": {
+        "kernelspec": {
+            "name": "python3",
+            "display_name": "unSkript (Build: 713)"
+        },
+        "language_info": {
+            "name": "python",
+            "file_extension": ".py",
+            "mimetype": "text/x-python",
+            "pygments_lexer": "ipython3"
+        },
+        "execution_data": {
+            "workflow_id": "4252287a-3ca0-4064-8f39-8f87c8a060fc",
+            "execution_id": "",
+            "notebook_id": "4a9e62f9-8466-4e14-8d05-74074aa53236.ipynb",
+            "tenant_id": "3c475b94-d99e-477e-bceb-7edf74500898",
+            "tenant_url": "https://us.app.unskript.io",
+            "proxy_id": "af32736b-fc75-4bdc-8785-8183cdeba8e1",
+            "parameters": [
+                "tag_key"
+            ],
+            "runbook_name": "delete user",
+            "environment_name": "Spotify_app",
+            "show_tool_tip": false,
+            "search_string": "",
+            "inputs_for_searched_lego": "",
+            "user_email_id": "doug.sillars@gmail.com",
+            "environment_type": "ENVIRONMENT_TYPE_UNSKRIPT_HOSTED"
+        },
+        "parameterSchema": {
+            "properties": {
+                "user_name": {
+                    "default": "FoobarAdmin",
+                    "description": "UserName to be deleted",
+                    "title": "user_name",
+                    "type": "string"
+                }
+            },
+            "required": [],
+            "title": "Schema",
+            "type": "object"
+        },
+        "parameterValues": {
+            "user_name": "FoobarAdmin"
+        }
+    },
+    "cells": [
+        {
+            "id": "3c35571b-9b95-4fa7-85ab-626fa913d77e",
+            "cell_type": "code",
+            "metadata": {
+                "actionNeedsCredential": false,
+                "actionSupportsIteration": false,
+                "actionSupportsPoll": false,
+                "deletable": false,
+                "editable": false,
+                "execution_data": {
+                    "last_date_success_run_cell": "2022-11-29T19:33:18.861Z"
+                },
+                "jupyter": {
+                    "source_hidden": true
+                },
+                "orderProperties": [],
+                "tags": [
+                    "unSkript:nbParam"
+                ],
+                "title": [
+                    "unSkript Internal"
```

Comment:
Please remove the first cell.

Reply:
fixed.  This was on both of them.  Perhaps the download from SAAS should remove this?